.github/workflows/build-and-push-image.yml

```diff
@@ -58,9 +58,10 @@ jobs:
         with:
           ref: ${{ github.ref }}
 
+      - run: echo "RELEASE_TAGNAME=${{ github.event.inputs.environment || "Dev" }}-`date +%Y-%m-%d`.${{ github.run_number }}" >> $GITHUB_ENV
```

Comment:
I've split this line out so that it sets the `env` correctly before trying to publish the `tag` in the next job

Reply:
Thanks. Sorry I know this is really fussy but for consistency please can you change 
${{ github.event.inputs.environment || "Dev" }}
to 
${{ github.event.inputs.environment || "dev" }}